Doc/c-api/import.rst

```diff
@@ -154,7 +154,7 @@ Importing Modules
    :class:`SourceFileLoader` otherwise.
 
    The module's :attr:`__file__` attribute will be set to the code object's
-   :c:member:`co_filename`.  If applicable, :attr:`__cached__` will also
+   :attr:`co_filename`.  If applicable, :attr:`__cached__` will also
```

Comment:
Isn't this the C member `co_filename` of `PyCodeObject`?

```suggestion
:c:member:`~PyCodeObject.co_filename`. If applicable, :attr:`__cached__` will also
```

Reply:
PyCodeObject is opaque. No one its C member is documented, and in general direct access to its members is not encouraged. On other hand, the code object in Python has documented attributes. The link in :attr:`co_filename` does not work yet, but this is a different issue.